### PR TITLE
[skip ci] Remove dummy-ansible-hosts inventory file

### PR DESCRIPTION
**Pull request by berendt**

### CODE REVIEW

**Comment by guits** [comment]
@berendt could you add a commit message, at least to explain your proposed change, please.

**Comment by berendt** [comment]
@guits Is this sufficient? I think a tidier root directory is prettier...

File: ceph-ansible.spec.in [hunk 1]
```diff
@@ -55,8 +55,7 @@ pushd %{buildroot}%{_datarootdir}/ceph-ansible
 popd
 
 %check
-# Borrowed from upstream's .travis.yml:
-ansible-playbook -i dummy-ansible-hosts test.yml --syntax-check
+ansible-playbook -i localhost, test.yml --syntax-check
```
Comment (leseb) [inline ceph-ansible.spec.in]: The syntax `localhost,` with the comma is a bit misleading even though it's correct. Perhaps we should just add `--connection=local` to the CLI and remove `-i localhost,`?

Thanks.

**Comment by guits** [comment]
No update for a while, closing this.
Feel free to reopen if needed.

**Pull request closed without merging**